wdio.conf.js

```diff
@@ -43,10 +46,21 @@ const config = {
       }],
     },
   },
+  terra: {
+    selector: '#wdio-test-wrapper',
+  },
 
    // Configuration for WebPackDevService
   webpackPort,
   webpackConfig,
+
```

Comment:
Lets not add this global refresh() hook. We don't need it at this point. Plus this is something we will want to remove from our webdriver.io configuration. If a test step needs to refresh after loading the browser, it should be specified within that test step that needs it.

Reply:
Removed: https://github.com/cerner/terra-clinical/pull/272/commits/3e968067da3a39875c1b528dd8615f112b61fd72